webcomponents/tool/src/main/frontend/js/rubrics/sakai-rubric.js

```diff
@@ -329,6 +329,27 @@ export class SakaiRubric extends RubricsElement {
       sakaiItemDelete.requestUpdate("rubric", this.rubric);
     }
   }
+
+ openEditWithKeyboard(e){
+    const classType = e.target.className;
+    const weighted = 'weighted';
```

Comment:
Why bother with a const for a string you're only going to use once? Just use the string.